### PR TITLE
API for downloading json files from s3

### PR DESCRIPTION
**Pull request by sweng66**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -144,8 +144,6 @@ services:
       - agr-literature
     volumes:
       - agr-logs:/logs
-    depends_on:
```
Comment (azurebrd) [inline docker-compose.yaml:147]: @sweng66 I think you don't want to remove that from the api docker-compose for main, but you did need that to run that on the devserver

**Review by azurebrd — COMMENTED** [review]
That's great @sweng66 !  
I think other than that postgres in docker compose getting removed, it's all good.
What are aiofiles and starlette ?  I don't see them in the changed files, so I'm wondering whether you tried them out and didn't use them, or whether they're used but don't need to be imported ?  Thanks !

File: docker-compose.yaml (hunk 1 above)
Comment (sweng66) [reply docker-compose.yaml]: @azurebrd  got it. I changed it back! thanks

**Comment by sweng66** [comment]
@azurebrd good catch! the fastapi needs aiofiles and starlette to be able to work with FileResponse properly.

**Review by valearna — APPROVED** [review]
Everything looks good, thanks @sweng66! I would pass 'bucketname' as an env variable and set it in the .env files but it's up to you if you want to do this now or later.

**Comment by sweng66** [comment]
@valearna Good suggestion!! I just added 'BUCKET_NAME' to .env and fixed the download.py to get bucketname from ENV.

**Comment by azurebrd** [comment]
Cool, thanks for explaining @sweng66 !   @valearna Passing the bucketname sounds fine to me, but it's always going to be agr-literature for the literature service, right ?  I guess this allows for in the future having a different root-level bucket that isn't for literature where the literature service can deposit and get files from ?

**Comment by valearna** [comment]
The bucket name could be defined in the code and doesn't need to be in the .env file if it's always going to be the same, but we could define it once (maybe in the api/config/GlobalConfig class?) and import it from the scripts instead of defining it inside each scripts.

**Comment by azurebrd** [comment]
That makes sense

**Comment by sweng66** [comment]
Thanks both for the explanation! @valearna @azurebrd   

Everything makes sense to me. The current fix is to use "agr-literature" as default bucket name if nothing is passed in from ENV for this variable so I will leave as is. Later we can 'globally' fix every program/module (that is using s3) to get the bucket name from a config file at the same time. Does this sound reasonable?

**Comment by azurebrd** [comment]
Cool, sounds good @sweng66 !  Let me try to revert main before merging this, so we probably shouldn't update this branch with main.

**Review by azurebrd — DISMISSED** [review]
Thanks @sweng66 and @valearna !

File: .env [hunk 2]
```diff
@@ -52,6 +52,8 @@ DEBEZIUM_KSQLDB_PORT=8088
 OKTA_CLIENT_ID=
 OKTA_CLIENT_SECRET=
 
+BUCKET_NAME=
```
Comment (valearna) [inline .env:55]: We should remove this variable from here

**Review by valearna — APPROVED** [review]
Everything looks good now, thanks @sweng66!

**Pull request merged**